<?xml version="1.0"?>


<!-- check: Streaming / VRPN Streaming Engine / On -->

<launch>
<!-- OptiTrack data -->
  <arg name="server" default="172.26.213.1"/>
  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883
      frame_id: 1
      broadcast_tf: true
      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
  </node>


<!-- Control  
	<node name="control_carro" pkg="control" type="control_carro.cpp" output="screen"></node>


<group ns="control_carro">
     <node pkg="control" name="control_carro" type="control_carro.cpp"/>
</group>-->
   
</launch>
